input/definitions/Resource/ValueSet-negation-reason-example.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "ValueSet",
+  "id": "negation-reason-example",
+  "url": "http://hl7.org/fhir/ValueSet/negation-reason-example",
+  "meta": {
+    "lastUpdated": "2024-07-01T07:07:38.434+11:00",
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
+      ]
+  },
+  "extension": [{
+    "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+    "valueCode": "fhir"
+  }],
+  "identifier": [
+    {
+      "system": "urn:ietf:rfc:3986",
+      "value": "urn:oid:2.16.840.1.113883.4.642.2.575"
+    }
```

Comment:
Was this identifier present in QICore? If so, it should be removed, this is a new extension, let the FHIR Extensions IG assign an OID for it